common/src/main/java/org/apache/uniffle/common/config/RssClientConf.java

```diff
@@ -40,7 +40,7 @@ public class RssClientConf {
   public static final ConfigOption<ShuffleDataDistributionType> DATA_DISTRIBUTION_TYPE = ConfigOptions
       .key("rss.client.shuffle.data.distribution.type")
       .enumType(ShuffleDataDistributionType.class)
-      .defaultValue(ShuffleDataDistributionType.NORMAL)
+      .defaultValue(ShuffleDataDistributionType.LOCAL_ORDER)
```

Comment:
We should use LOCAL_ORDER when we use Spark AQE. We shouldn't use it as default value.

Reply:
Make sense. Updated.